### PR TITLE
survey banner on safari fix

### LINKED ISSUE #1490: Survey banner displayed incorrectly on Safari
**Description**
Noticed a minor issue with the display of the survey on banner on Safari, it's much larger than it needs to be. 

**Steps to Reproduce**
View the home screen of Files in Safari (mobile or desktop) with a user who hasn't dismissed the survey banner

**Observed behavior**
IMG_613DACCA9FEF-1

**Mobile:**
 - OS: iOS 14
 - Browser: Safari
 - Device: iPhone 12 Mini
 -
**Desktop:**
 - OS: macOS 11.5.2
 - Browser: Safari

### PR DESCRIPTION
**Pull request by tanmoyAtb**
closes #1490

### CODE REVIEW

**Comment by asnaith** [comment]
The render link is not working for me, not sure why 🤔

**Comment by Tbaut** [comment]
> The render link is not working for me, not sure why thinking

idk either, manually relaunched it and it's live now.

File: packages/files-ui/src/Components/SurveyBanner.tsx [hunk 1]
```diff
@@ -91,7 +87,9 @@ const SurveyBanner = ({ onHide }: Props) => {
         className={classes.crossIconButton}
         onClick={onClose}
       >
-        <CrossIcon fontSize="small" />
+        <CrossIcon fontSize="small"
+          className={classes.icon}
```
Comment (Tbaut) [inline packages/files-ui/src/Components/SurveyBanner.tsx]:
```suggestion
        <CrossIcon
          fontSize="small"
          className={classes.icon}
```

**Review by Tbaut — APPROVED** [review]
This is looking good on FF, just a nit for the code.

**Review by RyRy79261 — APPROVED** [review]

**Review by FSM1 — APPROVED** [review]

**Comment by tanmoyAtb** [comment]
> The render link is not working for me, not sure why 🤔

Hey @asnaith, render links are working now, give this a check when you can!

**Comment by asnaith** [comment]
@tanmoyAtb 🤩 Looks great now on Safari in both macOS and iOS 👍

IMG_CA6A83A180E9-1

Screen Shot 2021-09-13 at 9 49 46 AM

**Review by asnaith — APPROVED** [review]

**Comment by tanmoyAtb** [comment]
> @tanmoyAtb 🤩 Looks great now on Safari in both macOS and Safari 👍
> 
> IMG_CA6A83A180E9-1
> 
> Screen Shot 2021-09-13 at 9 49 46 AM

Awesome! its good to go then !

**Pull request merged**